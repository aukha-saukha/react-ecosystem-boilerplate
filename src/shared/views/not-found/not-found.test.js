import { shallow } from 'enzyme';
import React from 'react';

import NotFound from './not-found.react';

describe('NotFound component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper).toMatchSnapshot();
  });
});
