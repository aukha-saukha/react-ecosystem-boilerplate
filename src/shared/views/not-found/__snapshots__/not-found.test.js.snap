// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component Renders correctly 1`] = `
<main
  className="font-size-16"
>
  Four oh! Four
</main>
`;
